### ISSUE TITLE
1.17: crash when typing in the control pane search

### ISSUE THREAD

**Issue by MeeniMc** [Siphalor/amecs#33]
Please use +1 emote if you want to indicate you also have this issue. Making a new comment sends emails and spam the author. 

## Version

MC 1.17, Amecs 1.3.4 21w16

## Replicate

1. Open the 'controls' menu
2. Click in the search box at the top
3. Game will crash (see log file below)
4. If game does not crash, type =
5. Game will crash

## Log

Log includes more addons (for notEnoughCrashes), but isssue can be replicated with nothing else than Amecs/Fabric API.

Note that the game's 'narrator' option is not active. 

```
---- Minecraft Crash Report ----
// You're mean.

Time: 6/14/21, 5:47 AM
Description: Narrating screen

java.lang.AbstractMethodError: Missing implementation of resolved method 'abstract java.util.List method_37025()' of abstract class net.minecraft.class_4265$class_4266.
	at Not Enough Crashes deobfuscated stack trace.(1.17+build.10)
	at net.minecraft.client.gui.widget.ElementListWidget$Entry.method_37024(ElementListWidget:94)
	at net.minecraft.client.gui.widget.ElementListWidget.appendNarrations(ElementListWidget:49)
	at net.minecraft.client.gui.screen.Screen.addElementNarrations(Screen:593)
	at net.minecraft.client.gui.screen.Screen.addScreenNarrations(Screen:576)
	at net.minecraft.client.gui.screen.narration.ScreenNarrator.buildNarrations(ScreenNarrator:34)
	at net.minecraft.client.gui.screen.Screen.narrateScreen(Screen:566)
	at net.minecraft.client.gui.screen.Screen.updateNarrator(Screen:553)
	at net.minecraft.client.render.GameRenderer.render(GameRenderer:890)
	at net.minecraft.client.MinecraftClient.render(MinecraftClient:1114)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:730)
	at net.minecraft.client.main.Main.main(Main:217)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:567)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.minecraft.class_4265$class_4266.method_37024(class_4265.java:94)
	at net.minecraft.class_4265.method_37020(class_4265.java:49)
	at net.minecraft.class_437.method_37056(class_437.java:593)
	at net.minecraft.class_437.method_37062(class_437.java:576)
	at net.minecraft.class_6385.method_37044(class_6385.java:34)
	at net.minecraft.class_437.method_37065(class_437.java:566)
	at net.minecraft.class_437.method_37071(class_437.java:553)

-- Screen details --
Details:
	Screen name: net.minecraft.class_458
Stacktrace:
	at Not Enough Crashes deobfuscated stack trace.(1.17+build.10)
	at java.lang.Thread.getStackTrace(Thread.java:1596)
	at net.minecraft.util.crash.CrashReportSection.trimStackTraceEnd(CrashReportSection:1023)
	at net.minecraft.util.crash.CrashReport.addElement(CrashReport:197)
	at net.minecraft.world.World.addDetailsToCrashReport(World:845)
	at net.minecraft.client.world.ClientWorld.addDetailsToCrashReport(ClientWorld:370)
	at net.minecraft.client.MinecraftClient.addDetailsToCrashReport(MinecraftClient:2399)
	at fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:28)
	at net.minecraft.client.MinecraftClient.modify$zcd000$onCrash(MinecraftClient:4599)
	at net.minecraft.client.MinecraftClient.run(MinecraftClient:749)
	at net.minecraft.client.main.Main.main(Main:217)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:567)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- Affected level --
Details:
	All players: 1 total; [class_746['MeeniPlus'/107, l='ClientLevel', x=6115.76, y=73.00, z=-758.20]]
	Chunk stats: 2025, 1560
	Level dimension: minecraft:overworld
	Level spawn location: World: (16,79,-144), Section: (at 0,15,0 in 1,4,-9; chunk contains blocks 16,0,-144 to 31,255,-129), Region: (0,-1; contains chunks 0,-32 to 31,-1, blocks 0,0,-512 to 511,255,-1)
	Level time: 97255 game time, 10059 day time
	Server brand: fabric
	Server type: Integrated singleplayer server
Stacktrace:
	at net.minecraft.class_638.method_8538(class_638.java:370)
	at net.minecraft.class_310.method_1587(class_310.java:2399)
	at fudge.notenoughcrashes.mixinhandlers.InGameCatcher.handleClientCrash(InGameCatcher.java:28)
	at net.minecraft.class_310.modify$zcd000$onCrash(class_310.java:4599)
	at net.minecraft.class_310.method_1514(class_310.java:749)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:226)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:146)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at org.multimc.onesix.OneSixLauncher.launchWithMainClass(OneSixLauncher.java:210)
	at org.multimc.onesix.OneSixLauncher.launch(OneSixLauncher.java:245)
	at org.multimc.EntryPoint.listen(EntryPoint.java:143)
	at org.multimc.EntryPoint.main(EntryPoint.java:34)

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, VanillaTweaks_r980949.zip

-- Last reload --
Details:
	Reload number: 1
	Reload reason: initial
	Finished: Yes
	Packs: Default, Fabric Mods, VanillaTweaks_r980949.zip

-- System Details --
Details:
	Minecraft Version: 1.17
	Minecraft Version ID: 1.17
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 16.0.1, GraalVM Community
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode, sharing), GraalVM Community
	Memory: 3445029320 bytes (3285 MiB) / 6442450944 bytes (6144 MiB) up to 6442450944 bytes (6144 MiB)
	CPUs: 12
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 5 3600 6-Core Processor              
	Identifier: AuthenticAMD Family 23 Model 113 Stepping 0
	Microarchitecture: Zen 2
	Frequency (GHz): 4.20
	Number of physical packages: 1
	Number of physical CPUs: 6
	Number of logical CPUs: 12
	Graphics card #0 name: Radeon RX 580 Series
	Graphics card #0 vendor: Advanced Micro Devices, Inc. (0x1002)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x67df
	Graphics card #0 versionInfo: DriverVersion=27.20.21002.112
	Memory slot #0 capacity (MB): 8192.00
	Memory slot #0 clockSpeed (GHz): 3.60
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 8192.00
	Memory slot #1 clockSpeed (GHz): 3.60
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 33742.68
	Virtual memory used (MB): 29048.55
	Swap memory total (MB): 17408.00
	Swap memory used (MB): 2056.64
	JVM Flags: 43 total; -XX:+UnlockExperimentalVMOptions -XX:+EnableJVMCIProduct -XX:-UnlockExperimentalVMOptions -XX:ThreadPriorityPolicy=1 -XX:+UseG1GC -XX:+ParallelRefProcEnabled -XX:MaxGCPauseMillis=16 -XX:+UnlockExperimentalVMOptions -XX:+UnlockDiagnosticVMOptions -XX:+EnableJVMCI -XX:+UseJVMCICompiler -XX:+DisableExplicitGC -XX:+AlwaysPreTouch -XX:G1NewSizePercent=30 -XX:G1MaxNewSizePercent=40 -XX:G1HeapRegionSize=8M -XX:G1ReservePercent=20 -XX:G1HeapWastePercent=5 -XX:G1MixedGCCountTarget=4 -XX:InitiatingHeapOccupancyPercent=15 -XX:G1MixedGCLiveThresholdPercent=90 -XX:G1RSetUpdatingPauseTimePercent=5 -XX:SurvivorRatio=32 -XX:+PerfDisableSharedMem -XX:MaxTenuringThreshold=1 -XX:UseAVX=2 -XX:SoftRefLRUPolicyMSPerMB=2000 -XX:+EagerXrunInit -XX:+UseXMMForObjInit -XX:+UseNewLongLShift -XX:+UseXmmI2D -XX:+UseXmmI2F -XX:+UseVectorCmov -XX:+OptoScheduling -XX:+OptoBundling -XX:+UseStringDeduplication -XX:+UseFastUnorderedTimeStamps -XX:UseSSE=4 -XX:+UseAES -XX:+UseAESIntrinsics -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms6144m -Xmx6144m
	Fabric Mods: 
		amecs: Amecs 1.3.4+mc.21w16a
		amecsapi: Amecs API 1.1.4+mc21w16a
		armor-visibility: Armor Visibility 3.0.0
		authme: Auth Me 1.5.0
		betterf3: BetterF3 1.1.5
		blue_endless_jankson: jankson 1.2.1
		boatiview: Boat Item View 0.0.2
		carpet: Carpet Mod 1.4.40
		carpet-autocraftingtable: Carpet-AutoCraftingTable 1.4.35
		carpet-extra: Carpet Extra 1.4.40
		carpet-tis-addition: Carpet TIS Addition 1.20.0+build.628
		chat-up: Chat Up! 2.0
		chunky: Chunky 1.2.86
		clear-skies: Clear Skies 1.5.66
		cleardespawn: Clear Despawn 1.17-fabric0.34.9-1.1.1
		cloth-basic-math: Cloth Basic Math 0.5.1
		cloth-config2: Cloth Config v4 5.0.34
		colorfulcut: Colorful Cut 1.0.0
		com_moandjiezana_toml_toml4j: toml4j 0.7.2
		creativeonepunch: Creative One-Punch 1.2
		crowdin-translate: CrowdinTranslate 1.3+1.17
		dark-loading-screen: Dark Loading Screen 1.6.4
		durabilityviewer: Giselbaers Durability Viewer 1.17-fabric0.34.9-1.10
		dynamicfps: Dynamic FPS 2.0.3
		easiercrafting: EasierCrafting 1.17-fabric0.34.9-1.6.7
		enchantedtooltips: Enchanted ToolTips 1.3.5-1.17
		enhanced_attack_indicator: Enhanced Attack Indicator 1.0.2+1.17
		fabric: Fabric API 0.35.1+1.17
		fabric-api-base: Fabric API Base 0.3.0+a02b44633d
		fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.1.0+1d383bb63d
		fabric-biome-api-v1: Fabric Biome API (v1) 3.1.11+c345aea83d
		fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.5+a02b44633d
		fabric-command-api-v1: Fabric Command API (v1) 1.1.1+a02b4463d5
		fabric-commands-v0: Fabric Commands (v0) 0.2.2+92519afa3d
		fabric-containers-v0: Fabric Containers (v0) 0.1.12+a02b44633d
		fabric-content-registries-v0: Fabric Content Registries (v0) 0.2.2+a02b44633d
		fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.1.5+be9da3103d
		fabric-dimensions-v1: Fabric Dimensions API (v1) 2.0.10+a02b44633d
		fabric-entity-events-v1: Fabric Entity Events (v1) 1.1.0+a02b44633d
		fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.8+a02b44633d
		fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.1+92519afa3d
		fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.6+a02b44633d
		fabric-item-api-v1: Fabric Item API (v1) 1.2.4+a02b44633d
		fabric-item-groups-v0: Fabric Item Groups (v0) 0.2.10+b7ab61213d
		fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.4+a02b44633d
		fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.2+36b77c3e3d
		fabric-language-kotlin: Fabric Language Kotlin 1.6.1+kotlin.1.5.10
		fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 1.4.4+a02b44633d
		fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.0.4+a02b44633d
		fabric-mining-levels-v0: Fabric Mining Levels (v0) 0.1.3+92519afa3d
		fabric-models-v0: Fabric Models (v0) 0.3.0+a02b44633d
		fabric-networking-api-v1: Fabric Networking API (v1) 1.0.11+b7ab61213d
		fabric-networking-blockentity-v0: Fabric Networking Block Entity (v0) 0.2.11+a02b44633d
		fabric-networking-v0: Fabric Networking (v0) 0.3.2+92519afa3d
		fabric-object-builder-api-v1: Fabric Object Builder API (v1) 1.10.9+b7ab61213d
		fabric-object-builders-v0: Fabric Object Builders (v0) 0.7.3+a02b44633d
		fabric-particles-v1: Fabric Particles (v1) 0.2.4+a02b44633d
		fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.7.10+e2961fee3d
		fabric-renderer-api-v1: Fabric Renderer API (v1) 0.4.3+676f40fa3d
		fabric-renderer-indigo: Fabric Renderer - Indigo 0.4.8+a02b44633d
		fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.0+a02b44633d
		fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.1.5+a02b44633d
		fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 0.1.13+a02b44633d
		fabric-rendering-v0: Fabric Rendering (v0) 1.1.2+92519afa3d
		fabric-rendering-v1: Fabric Rendering (v1) 1.6.0+a02b44633d
		fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.4.7+b7ab6121d5
		fabric-screen-api-v1: Fabric Screen API (v1) 1.0.4+198a96213d
		fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.1.8+a02b44633d
		fabric-structure-api-v1: Fabric Structure API (v1) 1.1.10+be9da3103d
		fabric-tag-extensions-v0: Fabric Tag Extensions (v0) 1.1.4+a02b44633d
		fabric-textures-v0: Fabric Textures (v0) 1.0.6+a02b44633d
		fabric-tool-attribute-api-v1: Fabric Tool Attribute API (v1) 1.2.12+b7ab61213d
		fabricloader: Fabric Loader 0.11.3
		gbfabrictools: GBfabrictools 1.3.4+1.17
		horsestatsvanilla: Horse Stats Vanilla 4.1.8
		inspecio: Inspecio 1.0.3+1.17
		itemscroller: Item Scroller 0.15.0-dev.20210608.211652
		jankson: Jankson 3.0.1+j1.2.0
		java: OpenJDK 64-Bit Server VM 16
		kyrptconfig: Kytpt Config 1.1.10-1.17
		lazydfu: LazyDFU 0.1.2
		libgui: LibGui 4.0.0-beta.4+1.17-rc1
		libninepatch: LibNinePatch 1.1.0
		litematica: Litematica 0.0.0-dev.20210608.211719
		loud_leads: Loud Leads 1.0.1+mc1.17
		malilib: MaLiLib 0.10.0+dev.22
		minecraft: Minecraft 1.17
		minihud: MiniHUD 0.19.0-dev.20210609.185508
		minitweaks: MiniTweaks 1.1.0
		modmenu: Mod Menu 2.0.2
		notenoughcrashes: Not Enough Crashes 3.3.0+1.17
		openloader: OpenLoader 2.0.1
		org_jetbrains_kotlin_kotlin-reflect: kotlin-reflect 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib: kotlin-stdlib 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk7: kotlin-stdlib-jdk7 1.5.10
		org_jetbrains_kotlin_kotlin-stdlib-jdk8: kotlin-stdlib-jdk8 1.5.10
		org_jetbrains_kotlinx_kotlinx-coroutines-core-jvm: kotlinx-coroutines-core-jvm 1.5.0
		org_jetbrains_kotlinx_kotlinx-coroutines-jdk8: kotlinx-coroutines-jdk8 1.5.0
		org_jetbrains_kotlinx_kotlinx-serialization-core-jvm: kotlinx-serialization-core-jvm 1.2.1
		org_jetbrains_kotlinx_kotlinx-serialization-json-jvm: kotlinx-serialization-json-jvm 1.2.1
		servertick: Server Tick 1.5
		sign_editor: Better Signs & Frames 0.8.1
		starlight: Starlight 1.0.0-RC2+fabric.e20b1d1
		tooltipfix: ToolTip Fix 1.0.2-1.16
		tweakeroo: Tweakeroo 0.10.0-dev.20210608.211857
		wthit: wthit 3.5.3
	Launched Version: MultiMC5
	Backend library: LWJGL version 3.2.2 build 10
	Backend API: Radeon RX 580 Series GL version 3.2.14830 Core Profile Forward-Compatible Context 21.4.1 27.20.21002.112, ATI Technologies Inc.
	Window size: 3840x2160
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: 
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	Graphics mode: fancy
	Resource Packs: vanilla, Fabric Mods, file/VanillaTweaks_r980949.zip
	Current Language: English (US)
	CPU: 12x AMD Ryzen 5 3600 6-Core Processor 
	Player Count: 1 / 8; [class_3222['MeeniPlus'/107, l='ServerLevel[Addon Tester]', x=6115.76, y=73.00, z=-758.20]]
	Data Packs: vanilla (incompatible), Fabric Mods
	Client Crashes Since Restart: 1
	Integrated Server Crashes Since Restart: 0
```

**Comment by Felix14-v2** [comment]
The same problem...

**Comment by ghost** [comment]
Can confirm, same issue.

Oddly enough, ~~it works in the controls menu from the main menu just fine~~ it worked once and caused a crash again on another attempt.

**Comment by supersaiyansubtlety** [comment]
Same issue

**Comment by MeeniMc** [comment]
Resolved in 1.3.4
